classes/controller/AdminController.php

```diff
@@ -1980,21 +1980,26 @@ private function getTabs($parentId = 0, $level = 0)
         $current_id = Tab::getCurrentParentId($this->controller_name ? $this->controller_name : '');
 
         foreach ($tabs as $index => $tab) {
-            if (!Tab::checkTabRights($tab['id_tab'])
-                || ($tab['class_name'] == 'AdminStock' && Configuration::get('PS_ADVANCED_STOCK_MANAGEMENT') == 0)
-                || $tab['class_name'] == 'AdminCarrierWizard') {
+            if (!Tab::checkTabRights($tab['id_tab']) ||
+                ($tab['class_name'] == 'AdminStock' && Configuration::get('PS_ADVANCED_STOCK_MANAGEMENT') == 0) ||
+                $tab['class_name'] == 'AdminCarrierWizard' ||
+                (Module::isInstalled('ps_mbo') && $tab['class_name'] == 'AdminModulesCatalog')
```

Comment:
I think this will be done by the MBO module after all

Reply:
That's the discussion we have with @Quetzacoalt91 , telling me there's no way to remove `tab` without editing core :/

Reply:
Why MBO module add his own "recommended modules" at first start? I agree with @eternoendless, we should hard code behavior in core related to modules.

Reply:
@mickaelandrieu That exactly what I said yesterday :P